### PR TITLE
Updated the java reference docs with cancelReceiveTimeout

### PR DESCRIPTION
**Pull request by prateekatknoldus**
References #31060

### CODE REVIEW

**Comment by prateekatknoldus** [comment]
Hi @johanandren @raboof. Can you please review this PR?

File: akka-docs/src/main/paradox/actors.md [hunk 1]
```diff
@@ -880,6 +880,8 @@ timeout there must have been an idle period beforehand as configured via this me
 Once set, the receive timeout stays in effect (i.e. continues firing repeatedly after inactivity
 periods). Pass in `Duration.Undefined` to switch off this feature.
```
Comment (johanandren) [inline akka-docs/src/main/paradox/actors.md:881]: I think this should go as well (even if that also works), better to document one way of doing it.
```suggestion
periods).
```

**Review by johanandren — COMMENTED** [review]
Note that there is a bug being fixed around `cancelReceiveTimeout` in https://github.com/akka/akka/pull/31059 that should be merged before we merge this doc update.

File: akka-docs/src/main/paradox/actors.md (hunk 1 above)
Comment (prateekatknoldus) [reply akka-docs/src/main/paradox/actors.md:881]: sure

**Review by johanandren — APPROVED** [review]
LGTM

**Review by octonato — APPROVED** [review]

**Pull request merged**